### PR TITLE
Feature/5217

### PR DESCRIPTION
**Pull request by jay-lee00**
classes for supporting new patient profile request.

### CODE REVIEW

File: SnapMD.ConnectedCare.ApiModels/PatientMedicalHistoryRequest.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{
+    public class PatientMedicalHistoryRequest
+    {
+        public int HistoryId { get; set; }
+        public int PatientId { get; set; }
+        public int? MedicalCondition1 { get; set; }
+        public int? MedicalCondition2 { get; set; }
+        public int? MedicalCondition3 { get; set; }
+        public int? MedicalCondition4 { get; set; }
+        public string PriorSurgery1 { get; set; }
```
Comment (lorddev) [inline SnapMD.ConnectedCare.ApiModels/PatientMedicalHistoryRequest.cs:13]: Please use an array for Surgeries instead. Json should look like:

```
"surgeries" : [
    "description" : string,
    "month" : .... etc.
]
```

There should be existing examples of this in the code, for example, in "Metadata" classes.

File: SnapMD.ConnectedCare.ApiModels/PatientMedicalHistoryRequest.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        public string PriorSurgery2 { get; set; }
+        public string PriorSurgery3 { get; set; }
+        public int? Surgery1Month { get; set; }
+        public int? Surgery2Month { get; set; }
+        public int? Surgery3Month { get; set; }
+        public int? Surgery1Year { get; set; }
+        public int? Surgery2Year { get; set; }
+        public int? Surgery3Year { get; set; }
+        public int? AllergicMedication1 { get; set; }
```
Comment (lorddev) [inline SnapMD.ConnectedCare.ApiModels/PatientMedicalHistoryRequest.cs:22]: Array, please

File: SnapMD.ConnectedCare.ApiModels/PatientProfileRequest.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        public string Height { get; set; }
+        public string Weight { get; set; }
+        public int? HeightUnit { get; set; }
+        public int? WeightUnit { get; set; }
+        public string Organization { get; set; }
+        public string Location { get; set; }
+        public int? OrganizationId { get; set; }
+        public int? LocationId { get; set; }
+
+        public string ValidateInputs()
```
Comment (lorddev) [inline SnapMD.ConnectedCare.ApiModels/PatientProfileRequest.cs:37]: Is there a unit test for the Validate method?

**Comment by lorddev** [comment]
@jay-lee00  please see comments, thanks!

**Comment by jay-lee00** [comment]
there is no unit test code, there is existing js code patientprofileBadges.viewmodel.js is using these fields.  the new classes match the used fields, and tested.

**Comment by lorddev** [comment]
@jay-lee00 Please create a brand new unit test in SnapMD.ConnectedCare.Sdk.Tests for the PatientProfileRequest.ValidateInputs() method.

**Comment by jay-lee00** [comment]
@lorddev unit test for PatientProfileRequest.ValidateInputs() method.added.

**Comment by lorddev** [comment]
@jay-lee00 please use the `MedicalHistoryProfile` class instead.

**Pull request closed without merging**